python/cli/prelude_cli/views/scm.py

```diff
@@ -107,11 +113,16 @@ def sync(controller, partner):
 def export(controller, type, output_file, limit, odata_filter, odata_orderby, partner):
     """ Export SCM data """
     with Spinner(description='Exporting SCM data'):
+        detect = DetectController(account=controller.account)
         export = ExportController(account=controller.account)
-        data = export.export_scm(export_type=type, filter=odata_filter, orderby=odata_orderby, partner=Control[partner] if partner else None, top=limit)
-        with open(output_file, 'wb') as f:
-            f.write(data)
-        return dict(status=True), f'Exported data to {output_file}'
+        job_id = export.export_scm(export_type=type, filter=odata_filter, orderby=odata_orderby, partner=Control[partner] if partner else None, top=limit)['job_id']
+        while (result := detect.get_background_job(job_id))['end_time'] is None:
```

Comment:
is this going to spam get background job, should we put a small sleep here?

Reply:
good idea. you are more polite than me